### PR TITLE
Add validation for review_state values to teacher_feedback

### PR DESCRIPTION
**Pull request by maureensturgeon**
Possible values for the new `review_state` column in teacher_feedbacks are null, "keepWorking" and "completed". This change adds validation to the column to ensure no other values are stored in the column.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/app/models/teacher_feedback.rb [hunk 1]
```diff
@@ -34,6 +34,13 @@ class TeacherFeedback < ApplicationRecord
   belongs_to :level
   belongs_to :teacher, class_name: 'User'
 
+  REVIEW_STATES = [
+    "KeepWorking",
```
Comment (jamescodeorg) [inline dashboard/app/models/teacher_feedback.rb]: Mostly curious, why did you choose PascalCase here?

Comment (maureensturgeon) [reply dashboard/app/models/teacher_feedback.rb]: Good question. I was thinking the performance_level was PascalCase but it's actually camelCase. For consistency I should probably use that. I'll update.

Comment (maureensturgeon) [reply dashboard/app/models/teacher_feedback.rb]: Thanks for pointing this out!

**Review by cforkish — APPROVED** [review]
LGTM!

**Pull request merged**